client/my-sites/domains/domain-management/dataviews/domains-dataviews.tsx

```diff
@@ -0,0 +1,77 @@
+import { DataViews, filterSortAndPaginate } from '@wordpress/dataviews';
+import { useI18n } from '@wordpress/react-i18n';
+import clsx from 'clsx';
+import { useEffect, useMemo, useState } from 'react';
+import { BulkUpdateNotice } from './bulk-update-notice';
+import { DomainData } from './types';
+import { useActions } from './use-actions';
+import { useDomainsDataViewsContext } from './use-context';
+import { getDomainId } from './use-domains';
+import './style.scss';
+import { useFields } from './use-fields';
+import useView, { getFieldsByBreakpoint } from './use-view';
+
+type Props = {
+	domains: DomainData[];
+	isLoading: boolean;
+	sidebarMode?: boolean;
+	selectedDomainName?: string;
+	openDomainPane?: ( domain: DomainData ) => void;
+};
+
+export const DomainsDataViews = ( {
+	domains,
+	isLoading,
+	sidebarMode,
+	selectedDomainName,
+	openDomainPane,
+}: Props ) => {
+	const { __ } = useI18n();
```

Comment:
Should we use `useTranslate` here too?

Reply:
Fixed here: https://github.com/Automattic/wp-calypso/pull/97677/commits/bf8860060b154ed44b272745308ba7343ccdf698